_static/author_bio.css

```
@@ -0,0 +1,30 @@
+.bio {
+    display: flex;
+    margin: 1em 0;
+}
+.photo {
+    max-width: 60px;
+    margin-right: 1em;
+}
+.photo__img {
+    width: 100%;
+    border-radius: 50%;
+}
+.bio-text__h4 {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-weight: bold;
+}
+.bio-text__p {
```

```suggestion
.bio-text__author-description {
```
Same comment as above.

---

Done!